src/components/pages/ProblemDetailedPage/index.tsx

```diff
@@ -1,54 +1,127 @@
-      <ProblemSubtitle subtitle={problemDetails.title} />
-      <ProblemSubtitle
-        color={secondarySubtitle}
-        subtitle={problemDetails.title}
-      />
-    </Wrapper>
+    <S.ProblemDetailedWrapper>
+      <S.GoToBackWrapper to="/">
+        <S.Arrow />
+        <S.GoToBackSpan>Powrót do listy problemów</S.GoToBackSpan>
+      </S.GoToBackWrapper>
+      <S.ProblemDetailedContent>
+        <S.EducationLevel>{problemDetails.level}</S.EducationLevel>
+        <S.TitleHeading>{problemDetails.title}</S.TitleHeading>
+        <S.ProblemCreationDetailsWrapper>
+          <S.ProblemCreationDetails>
+            {problemDetails.author}
+          </S.ProblemCreationDetails>
+          <S.ProblemCreationDetails>{formattedDate}</S.ProblemCreationDetails>
+        </S.ProblemCreationDetailsWrapper>
+        <S.Section>
+          <S.TagsWrapper>
+            {problemDetails.tags &&
+              problemDetails.tags.map((tagText: string) => (
+                <Tag text={tagText} key={tagText} />
+              ))}
+          </S.TagsWrapper>
+          <Button
+            $isPrimary={false}
+            withDownloadIcon={true}
+            $alignSelf="center"
+          >
+            Pobierz treść
+          </Button>
+        </S.Section>
+        <S.ProblemSection>
+          <ProblemSubtitle subtitle="Treść problemu" />
+          <S.ProblemSectionP>{problemDetails.description}</S.ProblemSectionP>
+          <S.ProblemSectionP>{problemDetails.question}</S.ProblemSectionP>
+        </S.ProblemSection>
+        <S.ProblemSection>
+          <ProblemSubtitle subtitle="Odpowiedź" />
+          <S.ProblemSectionP>{problemDetails.answer}</S.ProblemSectionP>
+        </S.ProblemSection>
+        <S.ProblemSection>
+          <ProblemSubtitle subtitle="Pobierz materiały dodatkowe" />
+          <S.ProblemSectionP>{problemDetails.resources}</S.ProblemSectionP>
+        </S.ProblemSection>
+        <S.ProblemSection>
+          <ProblemSubtitle subtitle="Propozycja wprowadzenia do zajęć" />
+          <S.ProblemSectionP>
+            {problemDetails.openingGuidance}
+          </S.ProblemSectionP>
+        </S.ProblemSection>
+        <S.ProblemExtensionHeading>
+          Rozszerzenie problemu
+        </S.ProblemExtensionHeading>
+        <S.ProblemSection>
+          <ProblemSubtitle
+            color={secondarySubtitle}
+            subtitle="Treść rozszerzenia"
+          ></ProblemSubtitle>
+          <S.ProblemSectionP>
+            {problemDetails.extension?.description}
+          </S.ProblemSectionP>
+          <S.ProblemSectionP>
+            {problemDetails.extension?.question}
+          </S.ProblemSectionP>
+        </S.ProblemSection>
+        <S.ProblemSection>
+          <ProblemSubtitle
+            color={secondarySubtitle}
+            subtitle="Odpowiedź"
+          ></ProblemSubtitle>
+          <S.ProblemSectionP>
+            {problemDetails.extension?.answer}
+          </S.ProblemSectionP>
+        </S.ProblemSection>
```

Comment:
Why the section "Pobierz materiały dodatkowe" is missing?

Reply:
Hmm, I can see this section. It is only in basic problem description (not extended). Just above "Propozycja wprowadzenia do zajęć"

![image](https://user-images.githubusercontent.com/51509583/148215865-5b188ce7-5c82-4ee3-91de-1a9cd0498e83.png)

Reply:
Great! I can see now. I don't know what I was looking at before.